charts/opentelemetry-collector/values.yaml

```diff
@@ -28,6 +28,7 @@ presets:
     # Enabling this writes checkpoints in /var/lib/otelcol/ host directory.
     # Note this changes collector's user to root, so that it can write to host directory.
     storeCheckpoints: false
+    maxRecombineLogSize: 1048576
```

Comment:
Is this number bytes or line numbers?

Reply:
Maybe worth having the link for this reference and the unit of measure as a comment on the chart.
WDYT?